tests/backup_helper.go

```diff
@@ -8540,3 +8542,56 @@ func GetUpdatedKubeVirtVMSpecForBackup(scheduledAppContextsToBackup []*scheduler
 	}
 	return nil
 }
+
+// SetPVCListBeforeRun will set the value of global variable pvcListBeforeRun with the PVC names
+func SetPVCListBeforeRun() error {
```

Comment:
How about creating a single function to return a list of PVCs and update the corresponding variable in its call sites? In this case BeforeSuite and AfterSuite.

```go
var _ = BeforeSuite(func() {
...

pvcListBeforeRun , err := GetPVCList()
log.FailOnError(err, "failed to list PVCs before run")

...
})
```

Reply:
@kphalgun-px I have used the global variables from backup helper and used the above approach as suggested. But I didn't re-declare the 'err' as it was already declared in the same block, and redeclaring was giving the error;

10:26:05  #28 23.68 ./backup_basic_test.go:278:7: err redeclared in this block
10:26:05  #28 23.68 	./backup_basic_test.go:252:8: other declaration of err
Please review.

Reply:
If it has already been declared, do we need to re-declare it? How about?

```go
var _ = BeforeSuite(func() {
...

pvcListBeforeRun, err = GetPVCList()
log.FailOnError(err, "failed to list PVCs before run")

...
})
```

Reply:
@kphalgun-px I believe that's how the current code is.